### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by 9th-unknown** [attardi/wikiextractor#216]
Hello everyone. I can't find an example of installing and using the library after its last update. Thanks

**Comment by Trannosaur** [comment]
Yeah it doesn't work any more.

**Comment by gregburman** [comment]
Yeah, I've also found that the latest set of commits have completely changed the project (WikiExtractor.py no longer exists and the README seems to be out-of-date wrt the latest changes). The big changes appear to have occurred on 22 July 2020 with [this commit](https://github.com/attardi/wikiextractor/commit/62bdbe6106148f8a870c7d1d9bc0d2536d174237): "Upgrade to Python 3.3+."

What I've done as an interim workaround is just rollback to the single [commit](https://github.com/attardi/wikiextractor/commit/e4abb4cbd019b0257824ee47c23dd163919b731b) from 29 March 2020 - this version appears to work and is more consistent with the README.

**Comment by iandoug** [comment]
Hi

Anyone got a working version with noob-proof installation instructions?

These python things install dependencies and then say they can't find them....

Thanks, Ian

**Comment by sandertan** [comment]
@iandoug  I think the pip-version is still working (`pip install wikiextractor`), but if you want to checkout the commit @gregburman suggested, you can checkout the commit and put it in a branch:
```bash
git checkout e4abb4cbd019b0257824ee47c23dd163919b731b -b stable
```

**Comment by iandoug** [comment]
@sandertan 
this one worked for my needs... puts it all in one plain text file with blank line between articles. I wrote PHP program to extract articles which met my needs and then put line breaks between the pargraphs.

https://github.com/apertium/WikiExtractor

Everything gets stripped including internal links, so you get things like ( , , , , ) or (; etc which may or not be a problem for you.

**Comment by sandertan** [comment]
Glad to hear you made it work. The March version with a fix to the category issue works for me.

**Comment by polm** [comment]
After finding #193 and generally having issues with this script, I started using the Wikipedia Cirrus dumps instead and would recommend anyone else do the same if possible. They have plaintext, raw wiki markup, category and redirect lists, and other useful data. 

https://dumps.wikimedia.org/other/cirrussearch/
https://meta.wikimedia.org/wiki/Data_dumps/Misc_dumps_format#Cirrus_search_index_dumps

One point of caution is that a few Cirrus dumps are incomplete, be sure to check you have the right number of articles in the dump you download. 

https://phabricator.wikimedia.org/T260986

**Comment by attardi** [comment]
I accepted a patch to make WikiExtractor into a PyPi package. But unfortunately somebody stole the WikiExtractor project name. I requested to get it back, but so far I am still waiting for the GitHub people to do it.
I put back the missing scripts.
Sorry for the inconvenience.
